packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx

```diff
@@ -4,27 +4,20 @@ import PropTypes from 'prop-types';
 import { Div100vhContainer, Icon, useOnClickOutside } from '@deriv/components';
 import { routes, isDesktop, isMobile, getActivePlatform, getPlatformSettings } from '@deriv/shared';
 import { BinaryLink } from 'App/Components/Routes';
-
 import 'Sass/app/_common/components/platform-dropdown.scss';
 
-const PlatformBox = ({ platform: { icon, title, description } }) => (
+const PlatformBox = ({ platform: { icon, description }, is_dark_mode }) => (
     <React.Fragment>
         <div className='platform-dropdown__list-platform-background' />
-        <Icon
-            data_testid='dt_platform_box_icon'
-            className='platform-dropdown__list-platform-icon'
-            icon={icon}
-            size={32}
-        />
 
-        <div className='platform-dropdown__list-platform-details'>
-            <p className='platform-dropdown__list-platform-title'>{title()}</p>
+        <div className='platform-switcher__dropdown'>
+            <Icon icon={is_dark_mode ? icon : `${icon}Dark`} height={42} width={150} />
```

Comment:
if it is dark mode you are using normal icon, and in the else condition you use dark mode in your condition. You need to change this.

Reply:
I also see some changes to some of our icons (Deriv Go and ...) but they are still the same. Can you explain what changes have you done ?

Reply:
@hirad-rewok I copied all the icons under the `brands` directory from an old PR into this PR during the cleanup